universal-login-commons/lib/core/utils/contracts/computeContractAddress.ts

```diff
@@ -6,3 +6,9 @@ export const computeCounterfactualAddress = (factoryAddress: string, publicKey:
   const hashedData = utils.solidityKeccak256(['bytes1', 'address', 'bytes32', 'bytes32'], ['0xff', factoryAddress, finalSalt, initCodeHash]);
   return utils.getAddress(`0x${hashedData.slice(26)}`);
 };
+
+export const computeContractAddress = (address: string, nonce: number) => {
```

Comment:
Tests missing. I would do a unit and integration here.

Reply:
✅